composer.json

```diff
@@ -16,22 +16,22 @@
         "sort-packages": true
     },
     "require": {
-        "php": "7.4.*",
+        "php": "^8.1",
         "ext-json": "*",
         "doctrine/annotations": "^1.10.0",
-        "symfony/config": "4.4.*|^5.0",
-        "symfony/dependency-injection": "4.4.*|^5.0",
-        "symfony/event-dispatcher": "4.4.*|^5.0",
-        "symfony/http-foundation": "4.4.*|^5.0",
-        "symfony/http-kernel": "4.4.*|^5.0",
-        "symfony/options-resolver": "^5.0"
+        "symfony/config": "6.1.*",
+        "symfony/dependency-injection": "6.1.*",
+        "symfony/event-dispatcher": "6.1.*",
+        "symfony/http-foundation": "6.1.*",
+        "symfony/http-kernel": "6.1.*",
+        "symfony/options-resolver": "6.1.*"
```

Comment:
Those requirements looks a bit too strong, and unnecessary.
I think php `>=8.0` and sf `^6.0` should be enough

Reply:
done